pkg/utils/kubernetes/resources/labels_test.go

```diff
@@ -0,0 +1,81 @@
+package resources
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/gateway-operator/pkg/consts"
+
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
+)
+
+func TestLabelObjectAsKonnectExtensionManaged(t *testing.T) {
+	for _, tt := range []struct {
+		name   string
+		input  map[string]string
+		output map[string]string
```

Comment:
Nit: this could be called `expected` to make this a bit more clear to the reader.